features/support/env.rb

```diff
@@ -5,6 +5,9 @@
 ###############################################################################
 # Aruba config and Cucumber hooks
 ###############################################################################
+BINARY_MAP = {
```

Comment:
I don't think you need that. Not if you, as intended,  use the actual binary name in the step